user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -927,9 +926,12 @@ function testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, app_id
   EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
   :Do(function(_,_)
     self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
-    { requestType = "PROPRIETARY", fileName = ptu_file_name})
+    { requestType = "PROPRIETARY", fileName = SystemFilesPath .. pts_file_name})
```

Comment:
@dboltovskyi, why you use path to the file. Regarding requirement "[HMI API] OnSystemRequest notification" HMI_API.xml fileName is file reference name.